src/components/DroughtHistory.vue

```diff
@@ -185,9 +176,32 @@ export default {
           .domain([new Date(timelineDates[0]), new Date(timelineDates[1])])
           .range([0, this.overlayHeight]);
 
-        // set up y axis
+        // set y-axis offset
         const yAxisOffset = this.mobileView ? 40: 45;
 
+        // Set up linear scale for chart width
+        const xScale = this.d3.scaleLinear()
+          .domain([0,100])
+          .range([0, this.overlayWidth])
+
+        // Add scroll to elements (only used for scroll navigation)
+        const scrollToSpot = this.svgChartDynamic.selectAll('scrollToSpot')
+          .data(this.scrollToDates)
+          .enter()
+          .append('rect')
+          .attr("id", d => "scrollStop-" + d.id)
+          .attr("class", "scrollToSpot")
+          .attr("x", yAxisOffset)
+          .attr("y", d => yScale(new Date(d.start)))
+          .attr("width", this.overlayWidth - yAxisOffset)
+          .attr("height", (d) => {
+            return yScale(new Date(d.end)) - yScale(new Date(d.start))
+          })
+          .attr("rx", 5)
```

Comment:
I like this with the grey and slightly rounded corners. I definitely wouldn't go any more rounded than this. I will update the viz with matching grey for now. On XD I had tried out light orange rectangles: Not sure which is better yet, until everything comes together, so I'll match the R generated viz to match this grey.

![image](https://github.com/USGS-VIZLAB/drought-timeline/assets/19958841/b0631daf-4113-4397-8a98-5d83814634a4)

Reply:
Agreed. I think we could even drop the rx a bit, maybe to 3?  I'll leave it as is for now, though